scripts/changelog.tmpl

```diff
@@ -66,3 +66,6 @@
 * {{. }}
 {{end -}}
 {{- end -}}
+
+**Docker Image Perma Link**:
```

Comment:
Can we just call this "Docker Image"? I mean it's an image name, not an actual link.

Reply:
Sure its both tho?

Reply:
I would expect a link to have a full form (i.e. start with `http(s)://...`). And while the names are eventually translated to links, you can't really directly download docker images like `alpine`, or `us-docker.pkg.dev/google-samples/containers/gke/hello-app:1.0` for example.